config/jobs/kubernetes-sigs/aws-load-balancer-controller/aws-alb-ingress-controller-presubmits.yaml

```diff
@@ -42,10 +42,10 @@ presubmits:
         - unit-test
         resources:
           limits:
-            cpu: 8
+            cpu: 6
             memory: 4Gi
           requests:
-            cpu: 8
+            cpu: 6
```

Comment:
Do we need the request? The pod scheduler could fail if it is unable to find the request. We can remove the CPU request, and rely on the limit.

Reply:
Request must match limit or the PR is rejected.

Reply:
I feel limit of 6 is still high, lets set it to 2, and we modify the UTs instead to decrease the resource usage.

Reply:
Let's set it high to unblock things, then once it's working we can experiment with non-blocking presubmits to find the real value.